frontend/.stylelintrc

```diff
@@ -15,96 +15,46 @@
         ]
       }
     ],
-    "at-rule-name-case": "lower",
-    "at-rule-name-newline-after": "always-multi-line",
-    "at-rule-name-space-after": "always",
     "at-rule-no-unknown": [
       true,
       {
         "ignoreAtRules": [
           "/^add\\-mixin$/",
           "/^define\\-mixin$/"
-        ] 
+        ]
       }
     ],
     "at-rule-no-vendor-prefix": true,
-    "at-rule-semicolon-newline-after": "always",
```

Comment:
Why are all these rules being removed?

Reply:
they are deprecated, Stylelint decided to go back to being just about correctness rules and leave stylistic choices up to tools like Prettier, see https://github.com/stylelint/stylelint/blob/main/docs/migration-guide/to-15.md